<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sale_Form</class>
 <widget class="QWidget" name="Sale_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="add_product_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Producto</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>451</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Fecha</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="date_label">
      <property name="text">
       <string>hoy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>451</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Empleado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="employees_combo"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>451</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Art√≠culos</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="detail_sale_table"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="total_line_edit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
